verticapy/plot.py

```diff
@@ -58,6 +58,7 @@
 from mpl_toolkits.mplot3d import Axes3D
 from matplotlib.colors import LinearSegmentedColormap
 from matplotlib.lines import Line2D
+import matplotlib.ticker as mticker
 import numpy as np
 
```

Comment:
Wouldn't it be confusing to have 2 plot.py files (there is on in the learn folder)? It might be better to add prefix or suffix to one of them; for example model_plot.py.

Reply:
Done;